Documentation/ceph-cluster-crd.md

```diff
@@ -1234,6 +1236,8 @@ spec:
     #rulesNamespace: rook-ceph
     #externalMgrEndpoints:
       #- ip: 192.168.39.182
+    # externalMgrServicesPort:
+    #   prometheus: 9283
```

Comment:
Is there another port we might need to expose? How about keeping it as a single settings and calling it `externalPrometheusPort`?

Reply:
I was initially heading that route but thought it'd be better to have it being generic so we don't add too many CRD entries. Now that it is a map we can add as many as we want.

Reply:
Reverted to a single option.